src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -0,0 +1,241 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Button, Divider, Link, Tab, TabList, Table, TableBody, TableCell, TableColumnDefinition, TableColumnSizingOptions, TableRow, Theme, createTableColumn, makeStyles, shorthands, teamsHighContrastTheme, useTableColumnSizing_unstable, useTableFeatures, webDarkTheme } from "@fluentui/react-components";
+import { useContext, useRef, useState } from "react";
+import { OpenFilled } from "@fluentui/react-icons";
+import { QueryResultContext } from "./queryResultStateProvider";
+import * as qr from '../../../sharedInterfaces/queryResult';
+import { useVscodeWebview } from '../../common/vscodeWebviewProvider';
+import SlickGrid, { SlickGridHandle } from "./slickgrid";
+
+const useStyles = makeStyles({
+	root: {
+		width: '100%',
+		height: '100%',
+		display: 'flex',
+		flexDirection: 'column',
+	},
+	ribbon: {
+		width: '100%',
+		display: 'flex',
+		flexDirection: 'row',
+		'> *': {
+			marginRight: '10px'
+		},
+	},
+	queryResultPaneTabs: {
+		flex: 1,
+	},
+	tabContent: {
+		...shorthands.flex(1),
+		width: '100%',
+		height: '100%',
+		display: 'flex',
+		...shorthands.overflow('auto'),
+	},
+	queryResultContainer: {
+		width: '100%',
+		height: '100%',
+		position: 'relative',
+	},
+	queryResultPaneOpenButton: {
+		position: 'absolute',
+		top: '0px',
+		right: '0px',
+	},
+	messagesContainer: {
+		width: '100%',
+		height: '100%',
+		flexDirection: 'column',
+		'> *': {
+			marginBottom: '10px'
+		},
+
+	},
+	messagesRows: {
+		flexDirection: 'row',
+		...shorthands.padding('10px'),
+		'> *': {
+			marginRight: '10px'
+		},
+	}
+});
+
+export const QueryResultPane = () => {
+	const classes = useStyles();
+	const state = useContext(QueryResultContext);
+	const webViewState = useVscodeWebview<qr.QueryResultWebviewState, qr.QueryResultReducers>();
+	webViewState;
+	var metadata = state?.state;
+
+	const getVscodeTheme = (theme: Theme) => {
+
+		switch (theme) {
+			case webDarkTheme:
+				return 'vs-dark';
+			case teamsHighContrastTheme:
+				return 'hc-black';
+			default:
+				return 'light';
+		}
+	};
+
+	getVscodeTheme;
```

Comment:
is this needed?

Reply:
no it's not, added it for debugging, will remove

Reply:
removed